src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -675,6 +675,58 @@ public async Task<IList<Check>> GetPullRequestChecksAsync(string pullRequestUrl)
             return statuses;
         }
 
+        /// <summary>
+        /// Retrieve the list of reviews on a PR.
+        /// </summary>
+        /// <param name="pullRequestUrl">Uri of pull request</param>
+        /// <returns>List of reviews.</returns>
+        public async Task<IList<Review>> GetPullRequestReviewsAsync(string pullRequestUrl)
+        {
+            (string accountName, string projectName, string repo, int id) = ParsePullRequestUri(pullRequestUrl);
+
+            JObject content = await this.ExecuteAzureDevOpsAPIRequestAsync(
+                    HttpMethod.Get,
+                    accountName,
+                    projectName,
+                    $"_apis/git/repositories/{repo}/pullRequests/{id}/reviewers",
+                    _logger);
+
+            JArray values = JArray.Parse(content["value"].ToString());
+
+            IList<Review> reviews = new List<Review>();
+            foreach (JToken review in values)
+            {
+                int vote = review["vote"].Value<int>();
+
+                ReviewState reviewState;
+
+                switch (vote)
+                {
+                    case 10:
```

Comment:
These are a bunch of magic numbers. Add something that explains what they mean.

Reply:
I'll link to the azdo documentation.  They are in fact...a bunch of magic numbers.